server/src/cg/group4/server/database/query/ResetPlayerData.java

```diff
@@ -12,24 +11,25 @@
 public class ResetPlayerData extends Query {
 
     /**
-     *
+     * The player ID to reset.
      */
     protected String cId;
 
+    /**
+     * Creates a new ResetPlayerData Query using the ID of that player.
+     * @param id The ID of the Player.
+     */
     public ResetPlayerData(final String id) {
         cId = id;
     }
 
     @Override
-    public Serializable query(DatabaseConnection databaseConnection) throws SQLException {
-        Statement statement = databaseConnection.query();
-        statement.executeUpdate("DELETE FROM Collectible WHERE GroupId = '" + cId + "'");
-        databaseConnection.commit();
-
-        statement.executeUpdate("DELETE FROM User WHERE Id = '" + cId + "'");
-        databaseConnection.commit();
-
-        statement.close();
+    public Serializable query(final Connection databaseConnection) throws SQLException {
```

Comment:
My opinion was that the user could remain in the database, but only the things that are 'obtained' need to be removed. Otherwise we'd have to immediately resubmit our player data in the database. This is something we can look at once we go over all the queries and screens.